pkg/asset/installconfig/alibabacloud/client.go

```diff
@@ -331,6 +323,23 @@ func (client *Client) GetAvailableZonesByInstanceType(instanceType string) ([]st
 	return zones, nil
 }
 
+func getEndpoint(productName string, regionId string) string {
```

Comment:
golint is failing because `regionId` should be `regionID`

Reply:
Have done